src/SimpleWebRequest.ts

```diff
@@ -626,7 +656,7 @@ export class SimpleWebRequest<TBody, TOptions extends WebRequestOptions = WebReq
                 // Some browsers error when you try to read status off aborted requests
             }
         } else {
-            statusText = 'Browser Error - Possible CORS or Connectivity Issue';
+            statusText = errorStatusText || 'Browser Error - Possible CORS or Connectivity Issue';
```

Comment:
Should the above usage of `errorStatusText` also come first? `this._xhr.statusText || errorStatusText` -> `errorStatusText || this._xhr.statusText` ?

Reply:
No - if we have a real status text - use that instead of the fallback.
But if we don't have the xhr at all - first use the provided value and then the fallback 
Basically priority is :
real status > passed status > generic 'Browser Error (...)' constant